src/views/common/Animation/AnimationControlButtonContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import { FunctionComponent, ReactNode, useMemo } from 'react'
```

Comment:
Buttons have been separated into their own files; the div that contains them is this `AnimationControlButtonContainer` and handles margins and spacing between them and stuff.

Basically I didn't want to repeat the code between the left and right control panels.